src/Compilers/CSharp/Portable/Symbols/Tuples/TupleErrorFieldSymbol.cs

```diff
@@ -35,16 +35,24 @@ public TupleErrorFieldSymbol(NamedTypeSymbol container, string name, int tupleFi
             _tupleFieldId = tupleFieldId;
         }
 
+        public override bool IsTupleField
+        {
+            get
+            {
+                return true;
+            }
+        }
+
         /// <summary>
         /// If this field represents a tuple element (including the name match), 
         /// id is an index of the element (zero-based).
         /// Otherwise, (-1 - [index in members array]);
         /// </summary>
-        public override bool IsTupleField
+        public int TupleFieldId
```

Comment:
Will see. The data is there, there is no reason to hide it.

---
In reply to: [62076508](https://github.com/dotnet/roslyn/pull/11047#discussion_r62076508) [](ancestors = 62076508)